modules/apps/data-engine/data-engine-rest-impl/src/test/java/com/liferay/data/engine/rest/internal/strategy/DefaultMapToDDMFormValuesConverterStrategyTest.java

```diff
@@ -57,6 +59,41 @@ public void setUp() {
 		_setUpLanguageUtil();
 	}
 
+	@Test
+	public void testCreateDDMFormFieldValueAsHashMap() {
+		DDMFormField ddmFormField = _createDDMFormField(
+			"field1", "document_library", true);
+
+		Map<String, Object> innerValues =
+			LinkedHashMapBuilder.<String, Object>put(
+				"en_US",
+				LinkedHashMapBuilder.put(
+					"key 1", "value 1"
+				).put(
+					"key 2", "value 2"
+				).put(
+					"key 3", "value 3"
+				).build()
+			).build();
+
+		List<DDMFormFieldValue> ddmFormFieldValues =
+			_defaultMapToDDMFormValuesConverterStrategy.
+				createDDMFormFieldValues(
+					LinkedHashMapBuilder.<String, Object>put(
+						"field1", innerValues
+					).build(),
+					ddmFormField, LocaleUtil.ENGLISH, null);
+
+		DDMFormFieldValue ddmFormFieldValue = ddmFormFieldValues.get(0);
+
+		JSONObject jsonObject = JSONFactoryUtil.createJSONObject(
+			(Map)innerValues.get("en_US"));
+		Value value = ddmFormFieldValue.getValue();
+
+		Assert.assertEquals(
+			jsonObject.toString(), value.getString(LocaleUtil.ENGLISH));
```

Comment:
Can we use JSONUtil.toString ?

Reply:
Hmm... Much better indeed :+1:

Reply:
Do you think it is better to format both outputs the `JSONUtil`'s way or is it okay to let it be?

Reply:
@jeyvison could you give your opinion here?

Reply:
The way it is is fine :)